### DISCUSSION TITLE
connect to another server ?

### DISCUSSION THREAD

**Discussion by GMib** [TwitchIO/TwitchIO#409] [Questions]
hi, Is there a simple way to connect to another IRC server? I would like to connect my bot to the fdgt.dev server to simulate cheers.

**Comment by chillymosh** [comment]
No, we connect to the official Twitch endpoint using IRC over websockets and not IRC directly.
If you can setup eventsub then you can simulate cheers using the CLI.

Just to note Pubsub and Eventsub both handle cheers but you can't emulate any pubsub events.

**Comment by GMib** [comment]
ok, thanks for response.
